### DISCUSSION TITLE
Log Retention Rules plugin doesn't work

### DISCUSSION THREAD

**Discussion by jeissonalex** [jongpie/NebulaLogger#455] [Q&A] [answered]
I have recently installed this unlocked plugin package in my Org sandbox and I notice that when running the plugin it is not working. In the logger settings I set: 

**Save Method: EVENT_BUS
Days to Retain Logs: 14
Platform Event Storage Location: CUSTOM_OBJECTS**

I tried to use the 'Sample Rule 1 - Error Logs' Retention Rule that it's on the package and the Retention Date of the logs isn't modified.
I did some debug on the **LogRetentionRulesPlugin.cls** apex class and I realize that on line 25:

`if (input.sobjectType != Schema.LogEntry__c.SObjectType)`

In the execution isn't accomplish the condition due to always the input.sojectType triggered is **LogEntryEvent__e** and condition is waiting a **LogEntry__c** and I assume that for this reason the plugin doesn't work.

Am I doing something wrong in the logger setting?
Does this pluggin only works with Save Method: SYNCHRONOUS_DML?

Thanks in advance for your help with this.

**Comment by jongpie** [comment, marked as answer]
This is a known issue - #354
